inlang/source-code/paraglide/paraglide-js-adapter-unplugin/package.json

```diff
@@ -39,23 +39,21 @@
 		"./dist"
 	],
 	"scripts": {
-		"build": "cross-env NODE_ENV=production node ./build.js && tsc",
-		"dev": "node ./build.js",
+		"build": "tsc",
+		"build:watch": "tsc -w",
```

Comment:
shouldnt build:watch this be dev?

Reply:
Oh yeah, that was copilot being goofy 😅 I'll fix it
I pinged you to make sure no longer bundling `@lix-js/client` is ok, I assume we're good there?